src/Arrays/vector_operations.jl

```diff
@@ -84,6 +83,19 @@ function ispermutation(u::AbstractVector{T}, v::AbstractVector)::Bool where {T}
     return true
 end
 
+function _in(x::AbstractVector{T}, itr) where {T}
+    return in(x, itr)
+end
+
+function _in(x::AbstractVector{T}, itr) where {T<:AbstractFloat}
+    for y in itr
+        if LazySets._isapprox(x, y)
+            return true
+        end
+    end
+    return false
```

Comment:
Does something like `return all(y -> LazySets._isapprox(x, y), itr)` not work?

Reply:
You probably mean `any(..)`. Yes, it is a good idea to make that change.

Reply:
I actually meant `all` with the negated condition, but that was still wrong here :smile: